File: SoloLearn/CMYK to RGB/CMYKtoRGB.rb
cmyk = []
(1..4).each { || cmyk += [ gets.chomp.to_f ] }
rgb = []
(0..2).each { |i| rgb += [ (255 * (1 - cmyk[i]) * (1 - cmyk[3])).round ] }
puts rgb.join(",")

File: SoloLearn/CMYK to RGB/CMYKtoRGB.swift
import Foundation

var cmyk: [Double] = []
for i in 0...3 {
    if let value = readLine(), let v = Double(value) {
        cmyk.append(v)
    }
}
var rgb: [String] = []
for i in 0...2 {
    rgb.append(String(Int(round(255 * (1 - cmyk[i]) * (1 - cmyk[3])))))
}
print(rgb.joined(separator: ','))